bedrock/firefox/templates/firefox/new.html

```diff
@@ -6,7 +6,22 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 {% block page_title %}{{_('Free Web Browser')}}{% endblock %}
-{% block page_desc %}{{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox today!')}}{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
```

Comment:
The page description is also changing here though? Will this likely also impact SEO on this page?